Tests/Unit/NetworkServer/ModuleConnectionHostTest.cs

```diff
@@ -115,6 +114,48 @@ public async Task On_Desired_Properties_Incorrect_Update_Should_Not_Update(strin
             Assert.Equal(facadeCode, localLoRaDeviceApiServiceBase.AuthCode);
         }
 
+        [Theory]
+        [InlineData(-100)]
+        [InlineData(0)]
+        [InlineData(5)]
+        [InlineData(400)]
+        [InlineData(1000)]
+        public async Task On_Desired_Properties_Correct_Update_Should_Update_Processing_Delay(int processingDelay)
+        {
+            var networkServerConfiguration = new NetworkServerConfiguration();
+            var classCMessageSender = Mock.Of<IClassCDeviceMessageSender>();
+            var loRaDeviceRegistry = Mock.Of<ILoRaDeviceRegistry>();
+            var loRaModuleClientFactory = Mock.Of<ILoRaModuleClientFactory>();
+
+            await using var moduleClient = new ModuleConnectionHost(networkServerConfiguration, classCMessageSender, loRaModuleClientFactory, loRaDeviceRegistry, loRaDeviceApiServiceBase, NullLogger<ModuleConnectionHost>.Instance, TestMeter.Instance);
+
+            Assert.Equal(400, networkServerConfiguration.ProcessingDelayInMilliseconds);
+
+            var input = JsonSerializer.Serialize(new
+            {
+                ProcessingDelayInMilliseconds = processingDelay,
+            });
+
+            await moduleClient.OnDesiredPropertiesUpdate(new TwinCollection(input), null);
+            Assert.Equal(processingDelay, networkServerConfiguration.ProcessingDelayInMilliseconds);
+        }
+
+        [Theory]
+        [InlineData("{ ProcessingDelayInMilliseconds: '' }")]
+        [InlineData("{ ProcessingDelay: 200 }")]
+        public async Task On_Desired_Properties_Incorrect_Update_Should_Not_Update_Processing_Delay(string twinUpdate)
```

Comment:
Shall we add tests covering also the unhappy path on init module for the delay property (like the one here for the properties update)?

Reply:
For sure we can add a test for module init with incorrect delay setting, but just to make sure we're on the same page, this will not be an 'unhappy path' scenario, since in this case the module init should succeed and the default delay would be used.

Reply:
Added new test in df7188bbce978d98caceb7410658ed48769a7b43

Reply:
Agree, meant "unhappy" as in "the config is incorrect". Thanks for the test